onnxruntime/core/providers/openvino/backends/backend_factory.cc

```diff
@@ -15,7 +15,7 @@ BackendFactory::MakeBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
                             GlobalContext& global_context,
                             const SubGraphContext& subgraph_context) {
   std::string type = global_context.device_type;
-  if (type == "CPU" || type.find("GPU") != std::string::npos ||
+  if (type == "CPU" || type.find("GPU") != std::string::npos || type == "VPUX" ||
```

Comment:
What if there's an iVPU and a dVPU (PCie)? Will we get device numbering then like VPUX.0? Can we also make this type.find('VPUX').  Also it'd be great to format these conditions neatly for better readability.